src/view.ts

```diff
@@ -115,15 +115,29 @@ export const divider = (): DividerBlock => {
     }
 };
 
-export const homeView = (blocks: (Block)[]) => {
+export const homeView = (avg: string, completedCount: number, items: string[]) => {
     return [
+        {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": `You've completed ${completedCount} action items`
+            }
+        },
+        {
+            "type": "section",
+            "text": {
+                "type": "mrkdwn",
+                "text": `On average, you take this long to complete action items: ${avg}`
+            }
+        },
```

Comment:
Should this section show up if we have completed 0 items?

Reply:
probably not, but given our audience I'd almost rather they know some feature is there. Next thing I plan on working on is re-arranging the home page and making it not look like garbage

Reply:
👍🏼